src/coreclr/nativeaot/System.Private.CoreLib/src/System/Buffer.NativeAot.cs

```diff
@@ -4,20 +4,19 @@
 using System.Diagnostics;
 using System.Runtime;
 using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
 
 using Internal.Runtime.CompilerServices;
 
 namespace System
 {
     public static partial class Buffer
     {
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        private static unsafe void ZeroMemoryInternal(void* b, nuint byteLength) =>
-            RuntimeImports.memset((byte*)b, 0, byteLength);
+        [LibraryImport(RuntimeHelpers.QCall, EntryPoint = "Buffer_Clear")]
+        private static unsafe partial void ZeroMemoryInternal(void* b, nuint byteLength);
```

Comment:
Move ZeroMemoryInternal and MemmoveInternal o src\libraries\System.Private.CoreLib\src\System\Buffer.cs under `#if !MONO`